src/allotropy/parsers/thermo_fisher_genesys30/constants.py

```diff
@@ -0,0 +1,13 @@
+""" Constants file for ThermoFisher Genesys 30 Adapter"""
+
+# Instrument software details
+GENESYS_SOFTWARE = "Genesys 30 instrument software"
+DISPLAY_NAME = "Thermo Fisher Genesys30"
+MODEL_NUMBER = "Genesys 30"
+BRAND_NAME = "Genesys"
+PRODUCT_MANUFACTURER = "Thermo Fisher Scientific"
+DEVICE_TYPE = "absorbance detector"
+
+# Error messages:
+UNSUPPORTED_FILE_FORMAT_ERROR = "Unsupported File Format. Expected .csv, .tsv. Actual: "
```

Comment:
Should this say "Unsupported file extension" just to be more explicit?

Reply:
Yes, it would be clearer and more explicit to use "Unsupported file extension" instead of "file format." I'll make this update. Thanks for the suggestion!